assets/downloads/jwt.html

```diff
@@ -334,6 +335,22 @@
           <option value="prod">Production</option>
         </select>
       </div>
+
+      <div>
+        <div class="form-label">
+          <div>
+            Please select the API Version for this JWT:
+          </div>
+          <div class="tooltip">
+            <img class="info-logo" src="https://dpc.cms.gov/assets/information-b8baf2e245c158d1de47193512ae79db36874c8f64793d1a47d9dbef70d86699.svg">
```

Comment:
SonarQube is asking for `alt` text for this image (for accessibility)

Reply:
The tooltips in the JWT tool are not displaying properly. I added an alt text tag for this tooltip img (please check), but there are others on this page. If this alt text is done correctly, I can include the alt text tag for all other tool tips as well.
![Screen Shot 2021-09-27 at 1 22 27 PM](https://user-images.githubusercontent.com/60108606/134956174-e45ff0cf-8e82-4b1f-933a-0ba372920080.png)

Thank you!